README.adoc

```diff
@@ -0,0 +1,5 @@
+# Eclipse MicroProfile
+
+This repository is no longer the active process for submitting feature ideas for Eclipse MicroProfile.
+
```

Comment:
Ken, where would we keep the approved proposals?  We'd still want a central location to have all of our MP proposals available, wouldn't we?